### DISCUSSION TITLE
cannot install pytgcalls

### DISCUSSION THREAD

**Discussion by Andinitri** [pytgcalls/pytgcalls#234] [Q&A]
pip install pytgcalls~=2.1.0
Collecting pytgcalls~=2.1.0
  Using cached pytgcalls-2.1.0-py3-none-any.whl.metadata (12 kB)
INFO: pip is looking at multiple versions of pytgcalls to determine which version is compatible with other requirements. This could take a while.
ERROR: Could not find a version that satisfies the requirement tgcalls==2.0.0 (from pytgcalls) (from versions: none)
ERROR: No matching distribution found for tgcalls==2.0.0``

help me, what is the solution? I have already use several ways and stil dont work to install pytgcalls

**Comment by TEAMUNKNOW** [comment]
pip install py-tgcalls




Try this
